test/args-test.cc

```diff
@@ -171,3 +171,16 @@ TEST(args_test, throw_on_copy) {
   }
   EXPECT_EQ(fmt::vformat("{}", store), "foo");
 }
+
+TEST(args_test, copy_constructor) {
+  auto store = fmt::dynamic_format_arg_store<fmt::format_context>();
+  store.push_back(fmt::arg("test1", "value1"));
+  store.push_back(fmt::arg("test2", "value2"));
+  store.push_back(fmt::arg("test3", "value3"));
+
+  auto store2 = fmt::dynamic_format_arg_store<fmt::format_context>(store);
```

Comment:
To get rid of the "useless cast" gcc warning:

```c++
  auto store2 = store;
```

Reply:
Fixed too.